<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_36">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_35">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Com ports</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSplitter" name="splitter_22">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter_15">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QLabel" name="taskForce">
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Force</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="portForce">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>PORT:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelPortForce">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </widget>
            <widget class="QComboBox" name="comLForce"/>
           </widget>
           <widget class="QSplitter" name="splitter_19">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QSplitter" name="splitter_14">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="conditionForce">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>CONDITION:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelConditionForce">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </widget>
            <widget class="QSplitter" name="splitter_16">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QPushButton" name="openBForce">
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
             <widget class="QPushButton" name="closeBForce">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_23">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter_24">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QLabel" name="taskLength">
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Length</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter_25">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="portLength">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>PORT:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelPortLength">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </widget>
            <widget class="QComboBox" name="comLLength"/>
           </widget>
           <widget class="QSplitter" name="splitter_26">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QSplitter" name="splitter_27">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="conditionLength">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>CONDITION:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelConditionLength">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </widget>
            <widget class="QSplitter" name="splitter_28">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QPushButton" name="openBLength">
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
             <widget class="QPushButton" name="closeBLength">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_29">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter_30">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QLabel" name="taskFrequency">
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Frequency</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter_31">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="portFrequency">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>PORT:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelPortFrequency">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </widget>
            <widget class="QComboBox" name="comLFrequency"/>
           </widget>
           <widget class="QSplitter" name="splitter_32">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QSplitter" name="splitter_33">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="conditionFrequency">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>CONDITION:</string>
              </property>
             </widget>
             <widget class="QLabel" name="labelConditionFrequency">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </widget>
            <widget class="QSplitter" name="splitter_34">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QPushButton" name="openBFrequency">
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
             <widget class="QPushButton" name="closeBFrequency">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Database</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QSplitter" name="splitter_12">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="NDB">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>NEW:</string>
            </property>
           </widget>
           <widget class="QSplitter" name="splitter_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="labelSet">
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Series:</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineSeries">
             <property name="text">
              <string>New Series</string>
             </property>
            </widget>
           </widget>
           <widget class="QLabel" name="DFD">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>DRAW FROM DATABASE:</string>
            </property>
           </widget>
           <widget class="QSplitter" name="splitter_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QLabel" name="drawFromDatabase">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Series</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comSeries">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </widget>
            <widget class="QSplitter" name="splitter_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QLabel" name="drawFromDatabase_2">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Experiment</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comExperiment">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Drawing</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLCDNumber" name="lcdN">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_11">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QSplitter" name="splitter_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="forwardButton">
             <property name="text">
              <string>forward</string>
             </property>
            </widget>
            <widget class="QPushButton" name="reverseButton">
             <property name="text">
              <string>reverse</string>
             </property>
            </widget>
           </widget>
           <widget class="QSplitter" name="splitter_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Frequency:</string>
             </property>
            </widget>
            <widget class="QDoubleSpinBox" name="spinFrequency"/>
           </widget>
           <widget class="QSplitter" name="splitter_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QRadioButton" name="radioNew">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>New</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QRadioButton" name="radioExisting">
             <property name="text">
              <string>Existing</string>
             </property>
            </widget>
           </widget>
           <widget class="QSplitter" name="splitter_37">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="drawB">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 170, 0);</string>
             </property>
             <property name="text">
              <string>START</string>
             </property>
            </widget>
            <widget class="QPushButton" name="stopDB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string>STOP</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QCustomPlot" name="wGraphic" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>240</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="labelCurrentForce">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current force:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="CurrentForceLCD"/>
      </widget>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="labelCurrentLength">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current length delta:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="CurrentLengthLCD"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="clickTheSettings"/>
  </widget>
  <action name="clickTheSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>to run the machine</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
